packages/node/test/integrations/http.test.ts

```diff
@@ -16,13 +16,14 @@ import { getDefaultNodeClientOptions } from '../helper/node-client-options';
 const NODE_VERSION = parseSemver(process.versions.node);
 
 describe('tracing', () => {
-  function createTransactionOnScope() {
+  function createTransactionOnScope(sendDefaultPii: boolean = true) {
     const options = getDefaultNodeClientOptions({
       dsn: 'https://dogsarebadatkeepingsecrets@squirrelchasers.ingest.sentry.io/12312012',
       tracesSampleRate: 1.0,
       integrations: [new HttpIntegration({ tracing: true })],
       release: '1.0.0',
       environment: 'production',
+      sendDefaultPii,
     });
```

Comment:
(optional) We could refactor this a bit so that `createTransactionOnScope` takes node client options as an argument which override a default (default being what's there now).

Reply:
Yes, good idea; that seems like a more sustainable solution

Reply:
[0241a8e](https://github.com/getsentry/sentry-javascript/pull/5344/commits/0241a8eae45d38173e9e8c6edd8c81a2eb9ee525)